p2p/starknet/ids.go

```diff
@@ -6,22 +6,27 @@ import (
 
 const Prefix = "/starknet"
 
-func HeadersPID() protocol.ID {
-	return Prefix + "/headers/0.1.0-rc.0"
+func HeadersPID(chainID string) protocol.ID {
+	return protocol.ID(Prefix + "/" + chainID + "/sync/headers/0.1.0-rc.0")
 }
 
-func EventsPID() protocol.ID {
-	return Prefix + "/events/0.1.0-rc.0"
+func EventsPID(chainID string) protocol.ID {
+	return protocol.ID(Prefix + "/" + chainID + "/sync/events/0.1.0-rc.0")
 }
 
-func TransactionsPID() protocol.ID {
-	return Prefix + "/transactions/0.1.0-rc.0"
+func TransactionsPID(chainID string) protocol.ID {
+	return protocol.ID(Prefix + "/" + chainID + "/sync/transactions/0.1.0-rc.0")
 }
 
-func ClassesPID() protocol.ID {
-	return Prefix + "/classes/0.1.0-rc.0"
+func ClassesPID(chainID string) protocol.ID {
+	return protocol.ID(Prefix + "/" + chainID + "/sync/classes/0.1.0-rc.0")
```

Comment:
Can we please use `utils.Network` type instead of a string for `*PID()` functions?